conf/metrics.properties.template

```diff
@@ -0,0 +1,9 @@
+
+# Enable StatsdSink for all instances by class name
+*.sink.statsd.class=org.apache.spark.metrics.sink.StatsdSink
```

Comment:
@hectorj2f  I think there's a typo in the classname. Should be "StatsDSink" (with an upper-case "D"), according to https://github.com/vidhyaarvind/spark-statsd (I also looked inside your jar and it had this class: "StatsDSink.class", also with "D")